components/Icons/ExternalLinkIcon.tsx

```diff
@@ -0,0 +1,23 @@
+type ExternalLinkIconProps = {
+    className?: string;
+    width?: number;
+    height?: number;
+};
+
+export const ExternalLinkIcon = ({
+    className = '',
+    width = 12,
+    height = 12,
+}: ExternalLinkIconProps): React.ReactElement => (
```

Comment:
`React.ReactElement` is a nice touch, typically I've not added react component return types in this project before

Reply:
I can't take the credit for this - I was just copying how another icon was done.